README.md

```diff
@@ -133,7 +133,54 @@ Be careful; while you can inspect and change even more as root, Bottlerocket's f
 Rather than a package manager that updates individual pieces of software, Bottlerocket downloads a full filesystem image and reboots into it.
 It can automatically roll back if boot failures occur, and workload failures can trigger manual rollbacks.
 
-Currently, you can update using a CLI tool, updog.
+The update process uses images secured by [TUF](https://theupdateframework.github.io/).
+For more details, see the [update system documentation](sources/updater/).
+
+There are several ways of updating your Bottlerocket instances:
+
+#### Update API
+
+The [Bottlerocket API](#api) allows you to update the instance from within the [control container](#control-container) or any [host container](#custom-host-containers) that has access to the Bottlerocket API.
+
+The update API is made up of a set of actions endpoints and a status endpoint to check the result of those actions.
+The behavior of the update API is governed by the [update settings](#updates-settings).
+
+In general, the update process looks like the following:
+
+Refresh list of available updates:
+```
+apiclient -u /actions/refresh-updates -m POST
+```
+
+Then check the list of available updates with:
+```
+apiclient -u /updates/status
```

Comment:
Maybe it would be beneficial to show the output of this command and explain how an update is the chosen_update?

Reply:
This section mainly serves as a high level overview of the steps to do a basic update. I'm somewhat reluctant to include the full structure of the update status here as I fear that might distract users from the commands to actually run. Following the steps, the user should be able to directly see the output and the referenced `chosen_update` field.

As for how an update is the `chosen_update`, that's determined by what is in the update settings which I reference a few lines above. I'll go ahead and add a reference to the `version-lock` setting to make it more clear.